src/site/markdown/server/manage-webadmin.md

```diff
@@ -435,6 +435,8 @@ This feature uses [Recipients rewrite table](/server/config-recipientrewritetabl
 the [RecipientRewriteTable mailet](https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java)
 to be configured.
 
+Note that email addresses needs to be composed of ASCII only characters. Mail addresses not matching this criteria will be rejected.
```

Comment:
more space after '+'?

Reply:
'+' is the marker for addition in Git. 

It's actually note contained in my proposal.

Reply:
lol

Reply:
There is more than one addresses so s/needs/need/.
Also I think "ASCII only characters" is weird. "only ASCII characters" sound better too me